working-with-openai/open-ai-api-endpoints.md

```diff
@@ -0,0 +1,156 @@
+# Using the OpenAI API
+
+## Goals
+
+Now that we know a bit about the OpenAI models, lets learn about the OpenAI endpoints. 
+
+Our goals for this lesson are to:
+- Recognize different OpenAI endpoints and how they are used
+- Write our first OpenAI request in Postman
+
+## Exploring OpenAI Endpoints
+
+In the last lesson, we looked at the different models associated with OpenAI. It may come as no surprise that each model maps to a specific endpoint we can use to access different AI functionalities. Below is a non-exhaustive list of OpenAI's endpoints and the models they map to:
+
+| Endpoint | Verb | Model(s) | Usage |
+| -------- | ---- | -------- | ----- |
+| https://api.openai.com/v1/chat/completions | POST | GPT 3.5 Turbo/GPT 4.0/ GPT 4.0 Turbo | Generating a response to a set of messages |
+| https://api.openai.com/v1/images/generations | POST | DALL-E | Generate a new image from a given prompt |
+| https://api.openai.com/v1/audio/speech | POST | TTS | Generate audio from input |
+| https://api.openai.com/v1/audio/transcriptions | POST | Whisper | Generate a transcription of audio input |
+| https://api.openai.com/v1/embeddings | POST | Embeddings | Creates a numerical  representation of text |
+| https://api.openai.com/v1/moderations | POST | Moderations | Checks to see if text could be potentially harmful | 
+| https://api.openai.com/v1/models | GET | None | Returns a list of OpenAI models | 
```

Comment:
I'd consider using code highlighting to make the Model names pop a bit more in that column, particularly to make the first one with several models a little easier to parse at a glance.

```suggestion
| Endpoint | Verb | Model(s) | Usage |
| -------- | ---- | -------- | ----- |
| https://api.openai.com/v1/chat/completions | POST | `GPT 3.5 Turbo`/ `GPT 4.0`/ `GPT 4.0 Turbo` | Generating a response to a set of messages |
| https://api.openai.com/v1/images/generations | POST | `DALL-E` | Generate a new image from a given prompt |
| https://api.openai.com/v1/audio/speech | POST | `TTS` | Generate audio from input |
| https://api.openai.com/v1/audio/transcriptions | POST | `Whisper` | Generate a transcription of audio input |
| https://api.openai.com/v1/embeddings | POST | `Embeddings` | Creates a numerical  representation of text |
| https://api.openai.com/v1/moderations | POST | `Moderations` | Checks to see if text could be potentially harmful | 
| https://api.openai.com/v1/models | GET | None | Returns a list of OpenAI models | 
```